src/index.js

```diff
@@ -207,6 +217,27 @@ export default function createTestHelpers(Kefir) {
     return log
   }
 
+  const {_onActivation, _onDeactivation} = Kefir.Observable.prototype
+
+  extend(Kefir, {
+    clearActiveObservables() {
+      Kefir.activeObservables = []
+    },
+  })
```

Comment:
Do we want to add the array + method to `Kefir` or can we just export them from `kefir-test-utils` directly? It would make it clearer where these methods come from and that they aren't built into Kefir but are provided by KTU.

Reply:
I _think_ that if we are to expose them directly from `kefir-test-utils` that we need to put them inside an object, something like this:

```
  const activeObservables = {
    observables: [],
    clear: function() { this.observables = [] }
  }

  return {
    activeObservables,
    ...
  }
```

am I right?

Reply:
Oh, the problem isn't ESM, it's because this is a function that returns the info rather than an export. The object approach seems good then.

Reply:
I've pushed it, not sure if you like the naming of the object keys. I noticed there is a failing test on node 16, I haven't looked into why it's failing yet.

Reply:
That seems to be a flaky test, I could only reproduce it the first time I ran the test, after that, I've ran tests more than 10x and they don't fail. Not sure if you have seen this behavior before or not.

Reply:
With #335 merged, the flaky test has been fixed. This is ready again.